src/meson.build

```diff
@@ -1042,11 +1042,17 @@ if get_option('tests').enabled()
   env.set('libs', meson.current_build_dir()) # TODO: Merge this with builddir after autotools removal
   HBSOURCES = []
   foreach f : hb_sources
+    if meson.version().version_compare('>=1.4.0')
+      f = f.full_path()
```

Comment:
The scripts that use these lists want relative paths, not absolute ones.

Reply:
Sorry, I didn't realize - since the tests all passed I assumed the scripts were able to handle them. I'll use fs.relative_to() to get back to the relative paths then.

Reply:
The scripts will convert the path to absolute path by joining them with base_srcdir, so full path would have simplified things if it were available in older versions.

Reply:
Or may be full path here is fine. It seems that `os.path.join(a, b)` will just return `b` if it is an absolute path.